test/e2e/activator_test.go

```diff
@@ -80,11 +82,6 @@ func TestActivatorOverload(t *testing.T) {
 		t.Fatalf("Error probing %s: %v", resources.Route.Status.URL.URL(), err)
 	}
 
-	deploymentName := rnames.Deployment(resources.Revision)
-	if err := WaitForScaleToZero(t, deploymentName, clients); err != nil {
```

Comment:
thinking out loud: I guess the question here is without the scale to zero, are we making the test too easy (ie because the activator wont have to buffer those 100 requests for 1-2 seconds until the first container comes up). There will still be quite a lot of buffering, though, because serviceSleep is 300ms and CC=1, so I think we're giving ourselves a little headstart compared to before, but not dramatically so.

Reply:
Right, I had the same thought. Practically, we should now queue 99 requests in the activator where before we queued 100.